public/server/db/gn.json

```
@@ -0,0 +1,4 @@
+{
+    "definition": "Good Night",
+    "alternatives": "Get Naked(used by couples or while sexting)"
```

I don't think the explanation in brackets is necessary haha.
```suggestion
    "alternatives": "Get Naked"
```